content/en/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -68,14 +68,14 @@ clusters and managed installations.
 There is a specific pattern for writing client programs that work well with
 Kubernetes called the *Controller* pattern. Controllers typically read an
 object's `.spec`, possibly do things, and then update the object's `.status`.
+A controller is a client of Kubernetes.
 
-A controller is a client of Kubernetes. When Kubernetes is the client and
-calls out to a remote service, it is called a *Webhook*. The remote service
-is called a *Webhook Backend*. Like Controllers, Webhooks do add a point of
-failure.
+When Kubernetes is the client and calls out to a remote service, 
+it is called a *Webhook*. The remote serviceis called a *Webhook Backend*. 
+Like Controllers, Webhooks do add a point of failure.
 
-In the webhook model, Kubernetes makes a network request to a remote service.
-In the *Binary Plugin* model, Kubernetes executes a binary (program).
+In the *Webhook* pattern, Kubernetes makes a network request to a remote service.
+In the *Binary Plugin* pattern, Kubernetes executes a binary (program).
```

Comment:
To me the use of "pattern" makes sense in "Controller pattern", but I think "model" fits better here. Let's see what other folks have to say :)

Reply:
Here is my reason for the modification.
> keep consistency with the subtitle (Extension Patterns)

Or we change the subtitle to `Extension Models`?

Reply:
I think patterns can include models. Taking a look at the closest definition of each (from a search):

- **pattern**: a form or model proposed for imitation
- **model**: a system or thing used as an example to follow or imitate

In that context, I take pattern to be larger themes and model(s) to be within patterns.